kokoro/ubuntu/release.cfg

```diff
@@ -11,6 +10,7 @@ action {
   # These are files we want to save once the build is done.
   define_artifacts {
     regex: "**/*.jar"
+    regex: "**/*.xml"
```

Comment:
might want to add `*.xz` as thats being created in the more recent tycho versions